apps/dashboard/src/components/billing/plans-row.tsx

```diff
@@ -1,160 +1,231 @@
+import React from 'react';
 import { Badge } from '@/components/primitives/badge';
 import { Card } from '@/components/primitives/card';
 import { Check } from 'lucide-react';
 import { ContactSalesButton } from './contact-sales-button';
 import { PlanActionButton } from './plan-action-button';
+import {
+  ApiServiceLevelEnum,
+  FeatureFlags,
+  FeatureFlagsKeysEnum,
+  FeatureNameEnum,
+  getFeatureForTierAsNumber,
+  getFeatureForTierAsText,
+  StripeBillingIntervalEnum,
+} from '@novu/shared';
 
 interface PlansRowProps {
-  selectedBillingInterval: 'month' | 'year';
-  currentPlan?: 'free' | 'business' | 'enterprise';
+  selectedBillingInterval: StripeBillingIntervalEnum;
+  currentPlan?: ApiServiceLevelEnum;
   trial?: {
     isActive: boolean;
   };
+  featureFlags: FeatureFlags;
 }
-
-interface PlanDisplayProps {
+interface PlanConfig {
+  name: string;
   price: string;
   subtitle: string;
   events: string;
+  features: string[];
+  actionType?: 'button' | 'contact';
 }
 
-function PlanDisplay({ price, subtitle, events }: PlanDisplayProps) {
-  return (
-    <div className="space-y-1">
-      <div className="flex items-baseline gap-1">
-        <span className="text-3xl font-bold tracking-tight">{price}</span>
-        <span className="text-muted-foreground text-sm font-medium">{subtitle}</span>
-      </div>
-      <span className="text-muted-foreground text-sm">{events}</span>
+const PlanFeature: React.FC<{ text: string }> = ({ text }) => (
```

Comment:
Please replace React.FC declarations and use simple functions as we follow the simple functional components pattern in the Dashboard.